config/sets/doctrine-code-quality.php

```diff
@@ -31,8 +30,6 @@
         TypedPropertyFromColumnTypeRector::class,
         TypedPropertyFromToOneRelationTypeRector::class,
         TypedPropertyFromToManyRelationTypeRector::class,
-
-        OrderByKeyToClassConstRector::class,
     ]);
```

Comment:
I wonder, why not add the rule here too?

Reply:
This specifically for doctrine 2.2+